connectivity/suite.go

```diff
@@ -187,6 +216,12 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			tests.PodToPod(),
 		)
 
+	// This policy allows port 8080 from client to echo (using label match expression, so this should succeed
+	ct.NewTest("client-egress-expression").WithPolicy(clientEgressToEchoExpressionPolicyYAML).
+		WithScenarios(
+			tests.PodToPod(),
+		)
```

Comment:
Why are there no Expectations on this test case?

Reply:
FWIW if you don't comment about this in the code, I'm pretty sure someone is going to look at it again in future and ask the exact same question. Omission is not always deliberate, so if you deliberately omit something then it's often easier to communicate by explicitly omitting it, eg by adding a line afterwards describing why this doesn't have any expectations.